gci/management/commands/filter_gci_non_issue_tasks.py

```diff
@@ -0,0 +1,36 @@
+import os.path
+
+from ruamel.yaml import YAML
+from django.core.management.base import BaseCommand
+
+from gci.gitorg import get_issue
+
+
+class Command(BaseCommand):
+    help = 'Extract the tasks which dont have an issue associated with them'
+
+    def handle(self, *args, **options):
+        non_issue_tasks = {}
+
+        yaml = YAML()
+
+        with open(os.path.join('_site', 'tasks.yaml')) as f:
+            tasks = yaml.load(f)
+
+            for task in tasks:
+                external_url = tasks[task]['external_url']
+                print('external issue for task is {}'.format(external_url))
```

Comment:
`external issue` -> `external url`.
The word issue here is confusing, or even wrong.

Reply:
Currently, link is logged first before task info is logged, e.g.
```
external issue for task is https://github.com/coala/gci-leaders/issues/30
task 6567626035691520 has a valid issue link
```
I think user experience would be better if log looks like this:
```
task 6567626035691520 has a valid issue link, external url is https://github.com/coala/gci-leaders/issues/30
```
i.e. put link and task info on the same line, and put task info before link url.